website/docs/docs/collaborate/explore-projects.md

```diff
@@ -51,75 +47,107 @@ By default, dbt Explorer shows the project's [applied state](/docs/dbt-cloud-api
+    - `resource_type:model [RESOURCE_NAME]` &mdash; Returns all models matching the name search
+    - `resource_type:metric,tag:nightly` &mdash; Returns metrics with the tag `nightly`
+- Use [graph operators](/reference/node-selection/graph-operators) (in the search bar) to select specific resources or a subset of the DAG. This can help narrow the focus on the resources that interest you. Examples:
+    - `+orders` &mdash; Returns all the upstream nodes of `orders`
+    - `+dim_customers,resource_type:source` &mdash; Returns all sources that are upstream of `dim_customers`
+- Use [set operators](/reference/node-selection/set-operators) (in the search bar) to select specific resources or a subset of the DAG. This can help narrow the focus on the resources that interest you. For example:
+    - `+snowplow_sessions +fct_orders` &mdash; Use space-delineated arguments for a union operation. Returns resources that are upstream nodes of either `snowplow_sessions` or `fct_orders`
 
 - [View resource details](#view-resource-details) by selecting a node (double-clicking) in the graph.
 - Click the List view icon in the graph's upper right corner to return to the main **Explore** page. 
 
-<LoomVideo id='4f85654b5cc443478299d2bb1e9211f6?t=0' />
+</expandable>
 
+### Example of lineage graph
 
-## Search for resources {#search-resources}
-With the search bar (on the upper left corner of the page or in a lineage graph), you can search with keywords or selectors. The resources that match your search criteria will display in a list (keyword search) or the lineage graph (selector search). 
+Example of exploring the `order_items` model in the project's lineage graph:
 
-### Search with keywords
-When searching with keywords, dbt Explorer searches through your resource metadata, such as resource name, and returns any matches. When searching for an exact column name, the results show all relational nodes containing that column in their schemas. If there's a match, a notice in the search result indicates the resource contains the specified column.
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-project-lineage-graph.png" width="100%" title="Example of full lineage graph" />
 
-You can then refine the search results using the filters in the panel on the right-hand side of the page. Use this panel to select specific resource types or tags to apply. You can filter on model properties (access or materialization type) under the **Models** option. 
+## Lenses
 
-### Search with selectors 
+The **Lenses** feature is available from your [project's lineage graph](#project-lineage). Lenses are like map layers for your DAG. Lenses make it easier to understand your project’s contextual metadata at scale, especially to distinguish a particular model or a subset of models. 
```

Comment:
Are we treating **Lenses** as a proper noun are is the bolded instance calling out a feature on screen? If it's the latter, can we add a blurb about where the user can find it and if it's the former should we capitalize in other areas?

Reply:
yeah, it's the latter. good flag for accessibility! 🙏